apps/ios/Shared/Model/SimpleXAPI.swift

```diff
@@ -365,6 +365,32 @@ func apiSendMessage(type: ChatType, id: Int64, file: CryptoFile?, quotedItemId:
     }
 }
 
+func apiCreateChatItem(noteFolderId: Int64, file: CryptoFile?, msg: MsgContent) async -> ChatItem? {
+    let chatModel = ChatModel.shared
+    let cmd: ChatCommand = .apiCreateChatItem(noteFolderId: noteFolderId, file: file, msg: msg)
+    var cItem: ChatItem? = nil
+    let endTask = beginBGTask({
```

Comment:
this is unnecessary here, it's not sending anything, can be done as other apis without background tasks